clients/src/main/java/org/apache/kafka/common/config/SslConfigs.java

```diff
@@ -89,11 +89,22 @@ public class SslConfigs {
     public static final String SSL_KEYSTORE_LOCATION_DOC = "The location of the key store file. "
         + "This is optional for client and can be used for two-way authentication for client.";
 
+    public static final String SSL_KEYSTORE_ALIAS_CONFIG = "ssl.keystore.alias";
+    public static final String SSL_KEYSTORE_ALIAS_DOC = "The Alias of key in the key store file. "
+            + "This is optional for client and can be used for two-way authentication for client.";
```

Comment:
Please provide right usage here. "This is config is used to pick named alias from the keystore to build the SSL engine and authenticate the client with broker. This is an optional config and used only when you have multiple keys in the keystore and you need to control which key needs to be presented to server."